src/site/stages/build/plugins/create-react-pages.js

```diff
@@ -29,6 +29,14 @@ function createTemporaryReactPages(options) {
               path: trimmedUrl,
               contents: new Buffer(`
               <nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
+                <div class="vads-l-grid-container">
+                  <div class="usa-alert usa-alert-warning">
```

Comment:
I didn't think about this initially, but that alert is pretty big. Maybe we should use the background only style? https://design.va.gov/components/alertboxes#background-color-only. That might be less intrusive, but still noticeable.